### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by nemccarthy** [graphql-go/relay#30]

**Comment by amkoehler** [comment]
I don't think it is. A couple small PRs (#29 and #27) have been sitting there a long time. With the reports of changes coming with Relay "2" I'm wondering if this library will stop working as is.
